hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/legacy/core/jproto/JKey.java

```diff
@@ -327,7 +327,7 @@ public static Key convertJKey(JKey jkey, int depth) throws InvalidKeyException {
                     .setThresholdKey(ThresholdKey.newBuilder().setKeys(keys).setThreshold(thd))
                     .build();
             return result;
-        } else if (jkey.hasKeyList()) {
```

Comment:
This threw NPE on line 331 without the if.
If there is no keylist, why are we getting the keylist?